vllm/attention/backends/flashinfer.py

```diff
@@ -113,8 +113,7 @@ def _get_decode_wrapper(self):
                 self.runner.parallel_config))
             num_kv_heads = self.runner.model_config.get_num_kv_heads(
                 self.runner.parallel_config)
-            use_tensor_cores = (num_qo_heads // num_kv_heads) not in \
-                (1, 2, 4, 8)
+            use_tensor_cores = num_qo_heads // num_kv_heads >= 4
```

Comment:
According to my experience, when `num_qo_heads // num_kv_heads == 4`,  `use_tensor_cores=False` is faster.
```suggestion
            use_tensor_cores = num_qo_heads // num_kv_heads > 4
```